README.md

```diff
@@ -74,51 +74,14 @@ cd cluster/images/hyperkube
 make VERSION=latest
 ```
 
-### Build CNI with plugins
-
-Firstly, you need to get a repo of CNI:
-
-```
-mkdir -p $GOPATH/src/github.com/containernetworking
-cd $GOPATH/src/github.com/containernetworking
-git clone git@github.com:containernetworking/cni.git
-```
-
-Then you can build CNI by doing:
+### Get CNI plugins
 
 ```
-cd cni
-./build
+get get -u github.com/containernetworking/plugins/plugins/main/bridge
```

Comment:
Did you mean `go get -u github.com/containernetworking/plugins/plugins/main/bridge`?